.ci/jenkins/dockerfiles/ubuntu_16_04.dockerfile

```diff
@@ -12,17 +12,32 @@ RUN apt-get update && apt-get -y --no-install-recommends install \
         cmake=3.5.1-1ubuntu3 \
         clang-3.9=1:3.9.1-4ubuntu3~16.04.2 \
         git=1:2.7.4-0ubuntu1.6 \
-        curl=7.47.0-1ubuntu2.12 \
+        curl=7.47.0-1ubuntu2.13 \
+        wget=1.17.1-1ubuntu1.5 \
         zlib1g=1:1.2.8.dfsg-2ubuntu4.1 \
         zlib1g-dev=1:1.2.8.dfsg-2ubuntu4.1 \
         libtinfo-dev=6.0+20160213-1ubuntu1 \
         unzip=6.0-20ubuntu1 \
         autoconf=2.69-9 \
         automake=1:1.15-4ubuntu1 \
+        ocl-icd-opencl-dev \
         libtool=2.4.6-0.1 && \
   apt-get clean autoclean && \
   apt-get autoremove -y
 
+# install the iGPU drivers copied into the container from the build context
+ARG opencl_url="https://github.com/intel/compute-runtime/releases/download"
+ARG opencl_version="19.29.13530"
+ARG igc_version="1.0.10-2306"
+ARG gmmlib_version="19.2.3"
+WORKDIR ${HOME}/intel-opencl
+RUN wget --no-check-certificate ${opencl_url}/${opencl_version}/intel-gmmlib_${gmmlib_version}_amd64.deb && \
```

Comment:
Are we adding these opencl libs to all images, even the ones which don't test opencl?

Reply:
Yes. The idea is to have one Docker image per OS that contains all dependencies for nGraph (we build it during CI anyway). I could split it to different images if You think it may cause problems.